### PR TITLE
Fix grimoire links.

### PR DESCRIPTION
**Pull request by jumarko**
Tested on a local build.

### CODE REVIEW

**Comment by puredanger** [comment]
For jbake, -b is the default so is not needed.

If you can remove that, I’ll merge.

**Comment by jumarko** [comment]
@puredanger 
I Installed it today via `brew install jbake`.
I tried `jbake` and it didn't do anything on my machine just prints help:
```
jbake
JBake v2.6.1 (2018-04-15 21:08:00PM) [http://jbake.org]

Usage: jbake
   or  jbake <source> <destination>
   or  jbake [OPTION]... [<value>...]

Options:
 <source>            : source folder of site content (with templates and assets), if not supplied
                       will default to current directory
 <destination>       : destination folder for output, if not supplied will default to a folder
                       called "output" in the current directory
 --reset             : clears the local cache, enforcing rendering from scratch (default: false)
 -b (--bake)         : performs a bake (default: false)
 -h (--help)         : prints this message (default: false)
 -i (--init)         : initialises required folder structure with default templates (defaults to
                       current directory if <value> is not supplied) (default: false)
 -s (--server)       : runs HTTP server to serve out baked site, if no <value> is supplied will
                       default to a folder called "output" in the current directory (default: false)
 -t (--template) VAL : use specified template engine for default templates (uses Freemarker if
                       <value> is not supplied)

```

**Comment by jumarko** [comment]
Updated.

File: content/api/api.adoc [hunk 1]
```diff
@@ -65,7 +65,7 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 
 == Other resources
 
-* http://grimoire.arrdem.com/[Grimoire] - community combined cheatsheet + examples
+* https://www.conj.io/[Grimoire] - community combined cheatsheet + examples
```
Comment (bbatsov) [inline content/api/api.adoc]: Btw, "community combined cheatsheet + examples" sounds a bit strange to me. If I didn't know what Grimoire was I would probably have struggled to figure this out.

Comment (puredanger) [reply content/api/api.adoc]: feel free to send follow-up pr

**Pull request merged**